README.md

```diff
@@ -1,2 +1,31 @@
-# StrathearnGeeks.github.io
-Static site for Strathearn Geeks
+# Strathearn Geeks website
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+## Overview
+
+This is the Strathearn Geeks website, as presented to Hugo. As such, there is an accompanying theme module [here](https://github.com/dwhweb/strath_geeks_theme) which contains the majority of the meat of the project in terms of HTML templates, Sass stylesheets, Javascript etc. This should be pulled down automatically as a dependency when you run the development server, or perform a build.
```

Comment:
I'm happy with this for now, but would it be possible to move the theme into the Strathearn Geeks area long term so that it may be maintained a bit easier in the future as required?